### PR TITLE
Bump julia@0.4

### PR DESCRIPTION
**Pull request by malmaud**

### CODE REVIEW

File: library/julia [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # maintainer: InfoSiftr <github@infosiftr.com> (@infosiftr)
 
+0.4: git://github.com/docker-library/julia@433ee54173ff0b8900045f5a81cad96b34558e9f
```
Comment (oeuftete) [inline library/julia]: Should there also be a `0.4.0` tag?

Comment (tianon) [inline library/julia]: Yeah, this whole file needs to be generated using the
"generate-stackbrew-library.sh" script in the julia repository.

Comment (malmaud) [inline library/julia]: Woops, didn't know that. I'll go ahead and do that.

Comment (tianon) [inline library/julia]: Cool!  Just give us a prod when you've got it updated (or let me know if you'd rather I just roll it into #1123). :+1:

**Comment by malmaud** [comment]
Alright, ready now.

**Comment by tianon** [comment]
Sweet, LGTM! :+1: Thanks!

Build test of #1121; 2a65e80053ddc901564d9b0b6e633190e922971b (`julia`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/2a65e80053ddc901564d9b0b6e633190e922971b/library/julia"
$ bashbrew build "$url"
Cloning julia (git://github.com/docker-library/julia) ...
Processing julia:0.4.0 ...
Processing julia:0.4 ...
Processing julia:latest ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing julia:0.4.0
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'julia-hello-world' [5/5]...
WARNING: deprecated syntax "println (" at /usr/src/julia/hello-world.jl:1.
Use "println(" instead.
passed
```

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**